### PR TITLE
Created Simple Text Summary application and CD

### PR DESCRIPTION
**Pull request by Lev0nid**

### CODE REVIEW

File: backend/process.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import torch
+from transformers import pipeline
+
+summarizer = pipeline(
```
Comment (SpirinEgor) [inline backend/process.py]: As in other languages, making something global isn't a good solution.

File: backend/process.py [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+    "summarization",
+    "pszemraj/long-t5-tglobal-base-16384-book-summary",
+    device=0 if torch.cuda.is_available() else -1,
```
Comment (SpirinEgor) [inline backend/process.py]: Conventionally, it is preferred to use "cuda" and "cpu" strings for devices
```python
"cuda" if torch.cuda.is_available() else "cpu"
```

File: main.py [hunk 3]
```diff
@@ -1,3 +1,26 @@
+from io import StringIO
 import streamlit as st
+from backend import process
 
-st.title('Work in progress')
+st.title('Simple Text Summary')
```
Comment (SpirinEgor) [inline main.py]: Don't make it global

**Review by SpirinEgor — CHANGES_REQUESTED** [review]
Nice 👍 
There are some comments that you need to take care of.

Also, there is no CD setup in this PR. I'm expecting either setup in this PR or separately.

**Comment by Lev0nid** [comment]
> Also, there is no CD setup in this PR. I'm expecting either setup in this PR or separately.

I've set up streamlit cloud. For instance, whenever this pull request is merged, the release version of the application will be updated automatically. The test version of the application is updated with each commit to the development branch.

File: backend/process.py (hunk 1 above)
Comment (Lev0nid) [reply backend/process.py]: Fixed

File: backend/process.py (hunk 2 above)
Comment (Lev0nid) [reply backend/process.py]: Fixed

File: main.py (hunk 3 above)
Comment (Lev0nid) [reply main.py]: Fixed

**Review by SpirinEgor — APPROVED** [review]

**Comment by SpirinEgor** [comment]
I don't see any settings for CD, therefore I can't validate it. Moreover, the link from Readme goes to failed app 🤷‍♂️

**Pull request merged**